Sources/Core/Filters/List/Views/ListFilterCell.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import FinniversKit
+
+final class ListFilterCell: CheckboxTableViewCell {
+    var isEnabled = true {
+        didSet {
+            isUserInteractionEnabled = isEnabled
+            checkboxImageView.isEnabled = isEnabled
+            overlayView.isHidden = isEnabled
+            bringSubviewToFront(overlayView)
+        }
+    }
+
+    private lazy var checkboxImageView = ListFilterImageView(withAutoLayout: true)
+
+    private lazy var chevronImageView: UIImageView = {
```

Comment:
I was a bit confused about this name. Maybe `alternativeAccessoryIconView`, or something a bit shorter :P

Reply:
I've managed to use default `accessoryView` from table view cell. Now it should even less confusing.